### PR TITLE
Reclaim quorum memory command

### PR DESCRIPTION
**Pull request by dcorbacho**
Depends on https://github.com/rabbitmq/rabbitmq-server/pull/2448

## Types of Changes
- [ ] Bug fix (non-breaking change which fixes issue #NNNN)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause an observable behavior change in existing systems)
- [ ] Documentation improvements (corrections, new content, etc)
- [ ] Cosmetic change (whitespace, formatting, etc)

## Checklist
- [x] I have read the `CONTRIBUTING.md` document
- [x] I have signed the CA (see https://cla.pivotal.io/sign/rabbitmq)
- [x] All tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in related repositories

### CODE REVIEW

File: lib/rabbitmq/cli/queues/commands/reclaim_quorum_memory_command.ex [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## License, v. 2.0. If a copy of the MPL was not distributed with this
+## file, You can obtain one at https://mozilla.org/MPL/2.0/.
+##
+## Copyright (c) 2007-2020 VMware, Inc. or its affiliates.  All rights reserved.
+
+defmodule RabbitMQ.CLI.Queues.Commands.ReclaimQuorumMemoryCommand do
+  alias RabbitMQ.CLI.Core.DocGuide
+
+  @behaviour RabbitMQ.CLI.CommandBehaviour
+  def scopes(), do: [:diagnostics, :queues]
```
Comment (michaelklishin) [inline lib/rabbitmq/cli/queues/commands/reclaim_quorum_memory_command.ex:11]: Does this really belong to `rabbitmq-diagnostics`?

Comment (lukebakken) [reply lib/rabbitmq/cli/queues/commands/reclaim_quorum_memory_command.ex:11]: `rabbitmq-queues` ???

Comment (michaelklishin) [reply lib/rabbitmq/cli/queues/commands/reclaim_quorum_memory_command.ex:11]: I'd only expose this to `rabbitmq-queues`, yes.

Comment (dcorbacho) [reply lib/rabbitmq/cli/queues/commands/reclaim_quorum_memory_command.ex:11]: Agreed, might have been some copy & paste from the status command

**Review by michaelklishin — APPROVED** [review]

**Comment by michaelklishin** [comment]
Backported to `v3.8.x`.

**Pull request merged**